tools_docker/Debian_Uwsgi/apache2.conf.sample

```diff
@@ -31,35 +31,43 @@
     # Can't be added in Location https://httpd.apache.org/docs/2.4/en/mod/mod_headers.html
     RequestHeader edit Destination ^https http early
 
-    # Proxying Webdav
+    # Proxy those paths to tracim web's with uwsgi protocol
+    <Location /agenda>
+        # ProxyPass uwsgi://localhost:8081/
+        # ProxyPassReverse uwsgi://localhost:8081/
+    </Location>
+    <Location /ui>
+        ProxyPass uwsgi://localhost:8081/
+        ProxyPassReverse uwsgi://localhost:8081/
+    </Location>
+    <Location /api>
+        ProxyPass uwsgi://localhost:8081/
+        ProxyPassReverse uwsgi://localhost:8081/
+    </Location>
+    <Location /custom_toolbox-assets>
+        ProxyPass uwsgi://localhost:8081/
+        ProxyPassReverse uwsgi://localhost:8081/
+    </Location>
+    <Location />
+        ProxyPass uwsgi://localhost:8081/
+        ProxyPassReverse uwsgi://localhost:8081/
+    </Location>
+
+
+    # Proxying Webdav to tracim_webdav
     <Location /webdav>
         # Preserving host is needed for working move/copy in webdav
         ProxyPreserveHost On
         #ProxyPass http://127.0.0.1:3030/webdav
         #ProxyPassReverse http://127.0.0.1:3030/webdav
```

Comment:
```suggestion
        # ProxyPass http://127.0.0.1:3030/webdav
        # ProxyPassReverse http://127.0.0.1:3030/webdav
```

Reply:
Adding same syntax like we have for agenda

Reply:
I changed those lines (and the sed entries in `entrypoint.sh`) for caldav as the `ProxyPass uwsgi://localhost:8081` line is exactly the same for all `Location` directive (thus leading to commenting all of them in case of `START_CALDAV=0`).
I also updated the webdav section to use the same way of making sections optional.

Reply:
Good.